data source nutanix_virtual_machine fails with "Invalid address to set: []string{"nic_list", "0", "is_connected"}"

**Describe the bug**
When trying to read a vm using the data source nutanix_virtual_machine hitting the following error,
`2019/05/07 06:45:49 [ERROR] root: eval: *terraform.EvalReadDataApply, err: data.nutanix_virtual_machine.myvm: Invalid address to set: []string{"nic_list", "0", "is_connected"}
2019/05/07 06:45:49 [ERROR] root: eval: *terraform.EvalSequence, err: data.nutanix_virtual_machine.myvm: Invalid address to set: []string{"nic_list", "0", "is_connected"}`

**Expected behavior**
Should be able to read the configuration and details of this attribute

**Logs**
`2019/05/07 06:45:49 [ERROR] root: eval: *terraform.EvalReadDataApply, err: data.nutanix_virtual_machine.myvm: Invalid address to set: []string{"nic_list", "0", "is_connected"}
2019/05/07 06:45:49 [ERROR] root: eval: *terraform.EvalSequence, err: data.nutanix_virtual_machine.myvm: Invalid address to set: []string{"nic_list", "0", "is_connected"}`

**Versions (please complete the following information):**
 - Mac OSX Mojave
 - Terraform version: 0.11.13
 - PE -5.10.1 , PC -5.10.1.1
 - provider.nutanix v1.0.1

--- a/nutanix/data_source_nutanix_virtual_machine.go
+++ b/nutanix/data_source_nutanix_virtual_machine.go
@@ -264,6 +264,10 @@ func dataSourceNutanixVirtualMachine() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+                                                "is_connected": {
+                                                        Type:     schema.TypeString,
+                                                        Computed: true,
+                                                },
 					},
 				},
 			},

--- a/nutanix/data_source_nutanix_virtual_machine.go
+++ b/nutanix/data_source_nutanix_virtual_machine.go
@@ -264,10 +264,10 @@ func dataSourceNutanixVirtualMachine() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
-                                                "is_connected": {
-                                                        Type:     schema.TypeString,
-                                                        Computed: true,
-                                                },
+						"is_connected": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
 					},
 				},
 			},

